sources/packages/backend/apps/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts

```diff
@@ -122,10 +119,7 @@ export class ProgramInfoRequestInstitutionsController extends BaseController {
     // assessment always available for submitted student applications).
     // PIR process happens only during original assessment.
     // Offering available only when PIR is completed.
```

Comment:
Are the comments at line 118-120 still relevant now? I thought it was describing the find original assessment code which was deleted.

Reply:
I kept it because I felt it was giving some additional context

Reply:
The code was reintroduced, so I believe that the comment is meaningful again.